.ci/.bump-stack-release-version.yml

```diff
@@ -49,3 +49,11 @@ projects:
     enabled: true
     labels: dependency
     title: "chore: update elastic stack release version"
+  - repo: e2e-testing
+    script: .ci/bump-stack-release-version.sh
+    branches:
+      - main
```

Comment:
Do we need main here? I expect to update the stale version for the last minor, only